components/dataViews/TransactionInfo.tsx

```diff
@@ -20,7 +20,7 @@ const TransactionInfo = (props: Props) => {
               <>
                 <li>
                   <label>Amount:</label>
-                  <div>{printableCoins(msg.value.amount, state.chain)}</div>
+                  <div>{printableCoins(msg.value.amount[0], state.chain)}</div>
```

Comment:
Amount can have multiple entiries. Not possible to create yet, but should be in the future. Doesn't `printableCoins` expect the array of coins?